fast64_internal/oot/oot_scene_room.py

```diff
@@ -672,9 +677,10 @@ def drawSceneHeaderProperty(layout, sceneProp, dropdownLabel, headerIndex, objNa
 			r.prop(sceneProp, "csTermIdx", text = "Index")
 			r.prop(sceneProp, "csTermStart", text = "Start Frm")
 			r.prop(sceneProp, "csTermEnd", text = "End Frm")
+		tempHeaderIndex = headerIndex - 4 if headerIndex is not None and headerIndex >= 4 else -1
```

Comment:
does this mean you can't add cutscenes to the "default alternate headers" aka child night, adult day and adult night?

Reply:
Thanks for pointing that out. When I was initially adding cutscene support for some reason I thought the existing fast64 code was set up to only permit an entrance cutscene and cutscenes in cutscene headers (not headers 1-3). So, I was intending to just follow this. However, I've looked again and it seems there's no reason the scene file format couldn't support cutscenes in headers 1-3, even if this is never used in game. So, I'll fix this to add that support.

Reply:
then you can get rid of `tempHeaderIndex` and just use `headerIndex` directly, have 0 stand for the main header instead of -1 standing for (all of?) the first four headers